docs/content/preview/yugabyte-platform/install-yugabyte-platform/prepare-environment/kubernetes.md

```diff
@@ -166,248 +166,170 @@ You need to perform the following steps:
-   4d33fcf3ee85: Already exists
-   60cbb698a409: Already exists
-   d90c5841d133: Pull complete
-   8084187ca761: Pull complete
-   47e3b9f5c7f5: Pull complete
-   64430b56cbd6: Pull complete
-   27b03c6bcdda: Pull complete
-   ae35ebe6caa1: Pull complete
-   9a655eedc488: Pull complete
-   Digest: sha256:286a13eb113398e1c4e63066267db4921c7644dac783836515a783cbd25b2c2a
-   Status: Downloaded newer image for quay.io/yugabyte/thirdparty-deps:latest
-   quay.io/yugabyte/thirdparty-deps:latest
-   ```
-
-   ```sh
-   docker pull postgres:11.5
-   ```
-
-   ```output
-   xxxxxx: Pulling from library/postgres
-   80369df48736: Pull complete
-   b18dd0a6efec: Pull complete
-   5c20c5b8227d: Pull complete
-   c5a7f905c8ec: Pull complete
-   5a3f55930dd8: Pull complete
-   ffc097878b09: Pull complete
-   3106d02490d4: Pull complete
-   88d1fc513b8f: Pull complete
-   f7d9cc27056d: Pull complete
-   afe180d8d5fd: Pull complete
-   b73e04acbb5f: Pull complete
-   1dba81bb6cfd: Pull complete
-   26bf23ba2b27: Pull complete
-   09ead80f0070: Pull complete
-   Digest: sha256:b3770d9c4ef11eba1ff5893e28049e98e2b70083e519e0b2bce0a20e7aa832fe
-   Status: Downloaded newer image for postgres:11.5
-   docker.io/library/postgres: 
-   ```
-
-   ```sh
-   docker pull prom/prometheus:v2.2.1
-   ```
-
-   ```output
-   Image docker.io/prom/prometheus:v2.2.1 uses outdated schema1 manifest format. Please upgrade to a schema2 image for better future compatibility. More information at https://docs.docker.com/registry/spec/deprecated-schema-v1/
-   aab39f0bc16d: Pull complete
-   a3ed95caeb02: Pull complete
-   2cd9e239cea6: Pull complete
-   0266ca3d0dd9: Pull complete
-   341681dba10c: Pull complete
-   8f6074d68b9e: Pull complete
-   2fa612efb95d: Pull complete
-   151829c004a9: Pull complete
-   75e765061965: Pull complete
-   b5a15632e9ab: Pull complete
-   Digest: sha256:129e16b08818a47259d972767fd834d84fb70ca11b423cc9976c9bce9b40c58f
-   Status: Downloaded newer image for prom/prometheus:
-   docker.io/prom/prometheus:
-   ```
-
-   ```sh
-   docker pull nginx:1.17.4
-   ```
-
-   ```output
-   1.17.4: Pulling from library/nginx
-   8d691f585fa8: Pull complete
-   047cb16c0ff6: Pull complete
-   b0bbed1a78ca: Pull complete
-   Digest: sha256:77ebc94e0cec30b20f9056bac1066b09fbdc049401b71850922c63fc0cc1762e
-   Status: Downloaded newer image for nginx:1.17.4
-   docker.io/library/nginx:1.17.4
-   ```
-
-   ```sh
-   docker pull janeczku/go-dnsmasq:release-1.0.7
-   ```
-
-   ```output
-   release-1.0.7: Pulling from janeczku/go-dnsmasq
-   117f30b7ae3d: Pull complete
-   504f1e14d6cc: Pull complete
-   98e84d0ba41a: Pull complete
-   Digest: sha256:3a99ad92353b55e97863812470e4f7403b47180f06845fdd06060773fe04184f
-   Status: Downloaded newer image for janeczku/go-dnsmasq:release-1.0.7
-   docker.io/janeczku/go-dnsmasq:release-1.0.7
-   ```
+  ```sh
+  docker pull quay.io/yugabyte/yugaware:${APP_VERSION}
+  docker pull quay.io/yugabyte/thirdparty-deps:latest
```

Comment:
This one is not used, we can remove it from here, as well as above output.
And remove it from other commands as well.